subsys/net/lib/lwm2m/lwm2m_rd_client.c

```diff
@@ -318,6 +332,8 @@ static int do_bootstrap_reply_cb(const struct coap_packet *response,
 		COAP_RESPONSE_CODE_DETAIL(code));
 
 	if (code == COAP_RESPONSE_CODE_CHANGED) {
+		/* remember the last reg time */
+		client.last_update = k_uptime_get();
```

Comment:
I don't think we should update the timer here. Bootstrapped is not yet registered.